### ISSUE TITLE
Bug with MultiMC Launcher?

### ISSUE THREAD

**Issue by Muzixs** [brandon3055/Draconic-Evolution#412]
I've been working on creating a modpack an noticed that D-E seems to be crashing as a result of using the MultiMC Launcher. This is simply an assumption; I added the mod before to the Pixelmon modpack on the Pixelmon Launcher with no issue.  I then reviewed the log several times trying to deduce the problem and noticed something about the core mod "CodeChickLib". I dont know what to make of it but I would like to point out that when I installed D-E using the Pixelmon launcher I did not add CodeChickLib, it worked just fine without. I only added it in use MultiMC in an attempt to exhaust all possible points of cause.

[Here's the client log](http://paste.ee/p/PtIFQ) when trying to load the pack with only D-E and the two respective required libs.

**Comment by covers1624** [comment]
Download the latest ccl from curse and place it in your mods folder, any ccl version in /mods/mcver gets delegated on startup.

**Comment by Muzixs** [comment]
Unfortunately I was unable to get it working through manual process. Instead I resorted to finding a modpack using it; copied the files and was able to jerry fix the issue that way. I'm not sure what I'm doing wrong on my part... I surmise it was a versioning issue. I noticed theres CCC, CCL, and CC. I was missing CC at the time so perhaps I overlooked a required lib. After it worked I dismissed the original issue from thought but will revisit it once I have more time to investigate.
